start.py

```diff
@@ -956,10 +957,17 @@ def sigterm_handler(_signo, _stack_frame):
 
     signal.signal(signal.SIGTERM, sigterm_handler)
 
-    service_backups()
-    set_up_nginx_files(m2ee)
-    complete_start_procedure_safe_to_use_for_restart(m2ee)
-    set_up_instadeploy_if_deploy_password_is_set(m2ee)
-    start_metrics(m2ee)
-    start_nginx()
-    loop_until_process_dies(m2ee)
+    try:
+        service_backups()
+        set_up_nginx_files(m2ee)
+        complete_start_procedure_safe_to_use_for_restart(m2ee)
+        set_up_instadeploy_if_deploy_password_is_set(m2ee)
+        start_metrics(m2ee)
+        start_nginx()
+        loop_until_process_dies(m2ee)
+    except Exception:
+        x = traceback.format_exc()
+        logger.error('Starting app container failed: %s' % x)
+        callback_url = os.environ.get('BUILD_STATUS_CALLBACK_URL')
+        if callback_url:
+            requests.put(callback_url, x)
```

Comment:
I think you want to raise the original exception here again at the end, via `raise`. Now only the error is logged but nothing crashes anymore, so Cloud Foundry doesn't know that the start failed.

Also, is there a good reason to try/except all these lines and not only around the `complete_start_procedure_safe_to_use_for_restart` call?